install.sh

```diff
@@ -40,7 +40,12 @@ esac
 
 # This will allow things to work without sudo
 crew_folders="bin cache doc docbook etc include lib lib$LIB_SUFFIX libexec man sbin share tmp var"
-for folder in $crew_folders ; do sudo chown -R "$(id -u)":"$(id -g)" "${CREW_PREFIX}"/"$folder" ; done
+for folder in $crew_folders
+do
+  if [ -d "${CREW_PREFIX}"/"${folder}" ] || [ -L "${CREW_PREFIX}"/"${folder}" ]; then 
+    sudo chown -R "$(id -u)":"$(id -g)" "${CREW_PREFIX}"/"${folder}"
+  fi
+done
 sudo chown "$(id -u)":"$(id -g)" "${CREW_PREFIX}"
```

Comment:
Is this line redundant?

Reply:
No. We need to chown just /usr/local too (non-recursively) too.